### PR TITLE
Better Hebrew support

### LINKED ISSUE #62: Translating all UI strings
Make sure every user presented string is translated for HE,EN

### PR DESCRIPTION
**Pull request by itaistopel**

### CODE REVIEW

**Comment by itaistopel** [comment]
שיניתי קצת חלק מהניסוחים

**Comment by itaistopel** [comment]
יש גם שגיאות / ניסוחים לא טובים / טעויות דקדוק באנגלית שכדאי מאוד לתקן :)

**Comment by itaistopel** [comment]
@svechinsky

File: app/src/main/java/com/example/coronadiagnosticapp/ui/views/FeelingQuestionView.kt [hunk 1]
```diff
@@ -40,10 +40,20 @@ class FeelingQuestionView : LinearLayout {
...
+            "Worse" -> res = context.getString(R.string.worse)
+        }
+        return res
+    }
+
     private fun addFeeling(feeling: GeneralFeeling) {
         val btn = EkRadioBtn(context).apply {
             id = feeling.ordinal
-            text = feeling.name.toLowerCase().capitalize()
+            text = getLocalizedFeeling(feeling.name.toLowerCase().capitalize())
```
Comment (SaparovNiv) [inline app/src/main/java/com/example/coronadiagnosticapp/ui/views/FeelingQuestionView.kt:56]: Is it working and saving data in English when you are on Heb and Eng ? 
and maybe you should create more generic method so when a user rather if the system on ENG or HEB 
do something which required saving data to the DB, you will get the data, 'translate' it to eng and only than save it to the db

File: app/src/main/java/com/example/coronadiagnosticapp/utils/MyApplication.kt [hunk 2]
```diff
@@ -8,6 +8,7 @@ import io.sentry.android.core.SentryAndroid
 class MyApplication : MultiDexApplication() {
     private lateinit var appComponent: AppComponent
     override fun onCreate() {
+
```
Comment (SaparovNiv) [inline app/src/main/java/com/example/coronadiagnosticapp/utils/MyApplication.kt:12]: Try to avoid unnecessary changes like this one

File: app/src/main/java/com/example/coronadiagnosticapp/data/network/NetworkDataSourceImpl.kt [hunk 3]
```diff
@@ -70,7 +70,7 @@ class NetworkDataSourceImpl @Inject constructor(val api: ApiServer) : NetworkDat
     }
 
     override suspend fun getQuestions(): List<JsonObject> {
-        val language = Locale.getDefault().language
+        val language = Locale.getDefault().language.replace("iw", "he")
```
Comment (SaparovNiv) [inline app/src/main/java/com/example/coronadiagnosticapp/data/network/NetworkDataSourceImpl.kt:73]: why replacing? isn't iw, and he are same ? I know that iw is the 'older' code for Hebrew

File: app/src/main/java/com/example/coronadiagnosticapp/data/network/NetworkDataSourceImpl.kt [hunk 4]
```diff
@@ -70,7 +70,7 @@ class NetworkDataSourceImpl @Inject constructor(val api: ApiServer) : NetworkDat
     }
 
     override suspend fun getQuestions(): List<JsonObject> {
-        val language = Locale.getDefault().language
```
Comment (eranCat) [inline app/src/main/java/com/example/coronadiagnosticapp/data/network/NetworkDataSourceImpl.kt:73]: Leave this one. it doesn't need extra work

File: app/src/main/java/com/example/coronadiagnosticapp/data/network/NetworkDataSourceImpl.kt (hunk 3 above)
Comment (svechinsky) [reply app/src/main/java/com/example/coronadiagnosticapp/data/network/NetworkDataSourceImpl.kt:73]: The server knows "he" not "iw"

File: app/src/main/java/com/example/coronadiagnosticapp/ui/views/FeelingQuestionView.kt (hunk 1 above)
Comment (svechinsky) [reply app/src/main/java/com/example/coronadiagnosticapp/ui/views/FeelingQuestionView.kt:56]: Yes, 100% 
The db needs to save English enum values

**Review by svechinsky — COMMENTED** [review]
Great job overall

File: app/src/main/java/com/example/coronadiagnosticapp/ui/views/FeelingQuestionView.kt (hunk 1 above)
Comment (eranCat) [inline app/src/main/java/com/example/coronadiagnosticapp/ui/views/FeelingQuestionView.kt:56]: That's a good point. we can add an enum field instead which represents the ResId int
and then just getString with it like so:
res = context.getString(feeling.resId)

**Review by eranCat — COMMENTED** [review]
👍 Just fix the other stuff (feeling thing)

**Comment by itaistopel** [comment]
@eranCat Have you fixed this enum thing?

**Comment by svechinsky** [comment]
@itaistopel @eranCat fix the string conflicts please and I'll merge
